.github/workflows/test.yaml

```diff
@@ -375,3 +375,49 @@ jobs:
 
       - name: run notebook
         run: jupyter nbconvert --to notebook --execute examples/text_classification/Text_Classification_using_Tensorflow_and_🔭_Galileo.ipynb
+
+  send-slack-notification:
+    runs-on: ubuntu-latest
+    if: ${{ always() }}
```

Comment:
I don't know how to read this but should this trigger always or only on failures?

Reply:
it should always trigger!